test/test_sdp.jl

```diff
@@ -243,6 +243,15 @@
                 #@fact x1==x2 --> true
             end
 
+            @testset "" begin
```

Comment:
Maybe `@testset "Issue #198" begin`?

Reply:
Whoops I forgot to name the test! I'm about to take a train but I'll fix that later today.

Reply:
Added it in 9392ca909cf2464513802484829fb29d9b037aeb.